chromadb/config.py

```diff
@@ -115,6 +115,7 @@ class Settings(BaseSettings):  # type: ignore
 
     is_persistent: bool = False
     persist_directory: str = "./chroma"
+    batch_ingest: bool = False
```

Comment:
is "background_ingest" a better term?

Reply:
It definitely better conveys what we're actually doing.